packages/network/tests/provider/rpc-mapper/methods/eth_getTransactionReceipt/eth_getTransactionReceipt.solo.test.ts

```
@@ -7,6 +7,23 @@ import {
+    } else if (typeof obj === 'object' && obj !== null) {
+        const newObj: Record<string, unknown> = {};
+        const objRecord = obj as Record<string, unknown>;
+        for (const key in objRecord) {
+            if (key !== 'blockHash' && key !== 'blockNumber') {
+                newObj[key] = removeBlockNumAndHashFields(objRecord[key]);
+            }
+        }
+        return newObj;
+    }
+    return obj;
+}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)